Javascript/Goodboy-Oldboy /script.js

```diff
@@ -1,18 +1,19 @@
-let dogYearBirth = 2014;
-let dogYearFuture = 2031;
-let shouldShowResultInDogYears = true;
+const dogYearBirth = 2014;
+const dogYearFuture = 2031;
+const shouldShowResultInDogYears = false;
 
-let dogAge;
-if (shouldShowResultInDogYears) {
-  dogAge = dogYearFuture - dogYearBirth;
-} else {
-  dogAge = (dogYearFuture - dogYearBirth) * 7;
-}
+const dogAgeInHumanYears = dogYearFuture - dogYearBirth;
+
+const dogAgeInDogYears = dogAgeInHumanYears * 7;
 
 if (shouldShowResultInDogYears) {
-  console.log(`Your dog will be ${dogAge} dog years old in ${dogYearFuture}`);
+  console.log(
+    `Your dog will be ${dogAgeInDogYears} dog years old in ${dogYearFuture}`
+  );
 } else {
-  console.log(`Your dog will be ${dogAge} human years old in ${dogYearFuture}`);
+  console.log(
+    `Your dog will be ${dogAgeInHumanYears} human years old in ${dogYearFuture}`
+  );
```

Comment:
No need to update it now, but as you use `dogAgeInDogYears` inside the `if else` you can definite it there, so if the condition is not true, you won't need to define it anyways.
```
const dogAgeInHumanYears = dogYearFuture - dogYearBirth;

if (shouldShowResultInDogYears) {
  const dogAgeInDogYears = dogAgeInHumanYears * 7;
  
  console.log(
    `Your dog will be ${dogAgeInDogYears} dog years old in ${dogYearFuture}`
  );
} else {
  console.log(
    `Your dog will be ${dogAgeInHumanYears} human years old in ${dogYearFuture}`
  );
}
```